webxrlayers-1.bs

```diff
@@ -532,8 +532,8 @@ The {{XRWebGLSubImage}} object is used during rendering of the layer.
 
 <pre class="idl">
 interface XRWebGLSubImage : XRSubImage {
-  readonly attribute WebGLTexture colorTexture;
-  readonly attribute WebGLTexture? depthStencilTexture;
+  [SameObject] readonly attribute WebGLTexture colorTexture;
```

Comment:
If we are re-using sub images in any capacity I think think this works, since the WebGLTexture will change frame-to-frame. Unless we're planning on having the underlying native texture swapped out while the JS wrapper stays the same, but that sounds questionable.

Reply:
> Unless we're planning on having the underlying native texture swapped out while the JS wrapper stays the same, but that sounds questionable.

yes, that will never happen:-)